Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -133,13 +133,15 @@ def handleProjectionEdgeCases(projection, data):
     # For mercator projection, latitude values of -90 or 90
     # transformation result in infinity values. We chose -85, 85
     # as that's the typical limit used by the community.
-    pname = projDict.get(projection._type, projection.type)
-    if (pname.lower() == "merc"):
-        lat = data.getLatitude()[:]
-        # Reverse the latitudes incase the starting latitude is greater
-        # than the ending one
-        if lat[-1] < lat[0]:
-            lat = lat[::-1]
+    ptype = projDict.get(projection._type, projection.type)
+    if (ptype.lower() == "merc"):
+        lat = data.getLatitude()
+        if isinstance(lat,cdms2.axis.TransientAxis):
```

Comment:
this only happend at the end when it works, otherwise there are a bunch of "flake8ed" commits and @aashish24 is not happy :wink: or I have to rebase, etc...